readme.md

```diff
@@ -135,6 +135,37 @@ const tasks = new Listr([
 It's also possible to return a `stream`. The stream will be converted to an `Observable` and handled as such.
 
 
+#### Skipping tasks
+
+<img src="media/skipped.png" width="255" align="right">
+
+At any point during execution a task can be skipped by invoking the `skip()` function on the task instance passed in to the task function.
+
+```js
+const tasks = new Listr([
+	{
+		title: 'Task 1',
+		task: () => Promise.resolve('Foo')
+	},
+	{
+		title: 'Can be skipped',
+		task: task => {
```

Comment:
Should we name it `ctx` instead? We might be able to pass more information later on.